modules/apps/oauth2-provider/oauth2-provider-jsonws/src/main/java/com/liferay/oauth2/provider/jsonws/internal/security/auth/verifier/OAuth2JSONWSAuthVerifier.java

```diff
@@ -85,14 +86,35 @@ public AuthVerifierResult verify(
 
 		AuthVerifierResult authVerifierResult = new AuthVerifierResult();
 
-		OAuth2Authorization oAuth2Authorization = _getOAuth2Authorization(
+		String accessTokenContent = _getAccessTokenContent(
 			accessControlContext);
 
+		if (accessTokenContent == null) {
+			authVerifierResult.setState(
+				AuthVerifierResult.State.NOT_APPLICABLE);
```

Comment:
Probably not necessery.

Reply:
I meant you don't have to explcity to set NOT_APPLICABLE state.
> authVerifierResult.setState(AuthVerifierResult.State.NOT_APPLICABLE);

Because AuthVerifierResult initializes with NOT_APPICABLE

Reply:
I didn't realize that. Amend as you see fit :)